torch/jit/_recursive.py

```diff
@@ -516,6 +530,15 @@ def ignore_overloaded(method_name):
         uniquer.add(name)
 
     stubs = []
+    for idx, fn in nn_module._forward_pre_hooks.items():
+        stubs.append(make_stub(fn))
+
+    for idx, fn in nn_module._forward_hooks.items():
+        stubs.append(make_stub(fn))
+
+    for idx, fn in nn_module._backward_hooks.items():
```

Comment:
Backward hooks aren't being used right ? we should probably error if they're set. "Backward Hooks NYI" etc

Reply:
Or should I drop this? Otherwise we may unnecessarily cause build failures. Users can fix their forward hooks but if we just error here, all they can do is to comment out backward hook registration in which case their eager mode runs will be impacted. We could ignore backward hooks entirely for now as we have been doing up until now.